src/style.css

```diff
@@ -1,3 +1,53 @@
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.logo {
+  margin-left: 50px;
+  padding-top: 10px;
+}
+
+.main_nav {
+  display: flex;
+  gap: 20px;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts. It's a good practice to specify a generic font family as a fallback in case the primary font is not available. For instance, `font-family: Roboto, Arial, sans-serif;` ensures that if Roboto is not available, Arial will be used, and if neither is available, the default sans-serif font of the user's browser will be used.